validator/static/validator/python/bin/validation_metadata.py

```diff
@@ -0,0 +1,120 @@
+import io
+import js
+import json
+from pyodide.http import open_url
+from validator.main_validator import PGSMetadataValidator
+from validator.request.connector import Connector, UnknownError, NotFound, Logger, ServiceNotWorking
+
+# Need a proxy for OLS as Pyodide causes a cross-origin issue with the OLS url
+OLS_URL = "https://ols-proxy-dot-pgs-catalog.appspot.com/ols-proxy/efo/%s"
```

Comment:
I think it would be better to move this variable in the app.yaml or the config file